src/App.js

```diff
@@ -1,12 +1,56 @@
-import React from 'react';
+import React, {useEffect, useState} from 'react';
 import './App.css';
+import axios from "axios";
+import Pokemoncard from "./components/Pokemoncard";
+
 
 function App() {
-  return (
-    <div>
-      Begin hier met de opdracht!
-    </div>
-  );
+
+    const [pokemonData, setPokemonData] = useState([]);
```

Comment:
Uiteindelijk is dit niet van het type array, ik zou dan als default waarde `undefined` of `{}` aanhouden